README.md

```diff
@@ -50,10 +50,23 @@ perform authenticated HTTP requests.
 
     signer = atlassian_jwt_auth.create_signer('issuer', 'issuer/key', private_key_pem)
     response = requests.get(
-        'https://your-url'
+        'https://your-url',
         auth=JWTAuth(signer, 'audience')
     )
 ```
+One can also use `JWTAuth` to autenticate `aiohttp` requests:
+
+```python
+    import atlassian_jwt_auth
+    from atlassian_jwt_auth.contrib.requests import JWTAuth
+
+    signer = atlassian_jwt_auth.create_signer('issuer', 'issuer/key', private_key_pem)
+    response = await aiohttp_client_session.get(
```

Comment:
Adding code for initializing a ClientSession session would make the example less illustrative